packages/block-editor/src/store/defaults.js

```diff
@@ -37,64 +36,6 @@ export const PREFERENCES_DEFAULTS = {
  */
 export const SETTINGS_DEFAULTS = {
 	alignWide: false,
-	colors: [
```

Comment:
The removal here means someone using the block editor package without providing a color palette may not work properly, did we check on the playground? Maybe it's fine but it's an npm package breaking change we should probably mention on the changelog.

Reply:
We are going to have the WordPress defaults that the server needs to be aware of ( to generate CSS vars for example). Having defaults on the block editor inscreses the maintainability efforts. I guess we can remove the block editor defaults, I added a braking change note.